integration-tests/src/test/java/tachyon/security/MasterClientAuthenticationIntegrationTest.java

```diff
@@ -161,6 +161,8 @@ public void customAuthenticationDenyConnectTest() throws Exception {
             mLocalTachyonCluster.getMasterTachyonConf());
     Assert.assertFalse(masterClient.isConnected());
     masterClient.connect();
+    // should never reach here, but this silences the resource not closed warning
+    masterClient.close();
```

Comment:
Intuitively, it makes more sense to use an annotation to suppress the warning that introducing unreachable code. But I don't have a strong preference either. @calvinjia do you have a preference?